components/definition/stimulation/NetPyNEStimulationTarget.js

```diff
@@ -1,29 +1,60 @@
 import React, { Component } from 'react';
-import SelectField from 'material-ui/SelectField';
+
 import MenuItem from 'material-ui/MenuItem';
 import TextField from 'material-ui/TextField';
+import SelectField from 'material-ui/SelectField';
+
 import Tooltip from 'material-ui/internal/Tooltip';
-import FlatButton from 'material-ui/FlatButton';
 import Toggle from 'material-ui/Toggle';
+
 import IconMenu from 'material-ui/IconMenu';
+import FontIcon from 'material-ui/FontIcon';
+import FlatButton from 'material-ui/FlatButton';
 import RaisedButton from 'material-ui/RaisedButton';
+
+import CardText from 'material-ui/Card';
+import { BottomNavigation, BottomNavigationItem } from 'material-ui/BottomNavigation';
+
 import clone from 'lodash.clone';
+
 import Utils from '../../../Utils';
 import NetPyNEField from '../../general/NetPyNEField';
+import ListComponent from '../../general/List';
+import Conditions from './Conditions';
 
+import Conds from 'material-ui/svg-icons/maps/local-offer';
+import StimTarget    from 'material-ui/svg-icons/action/reorder';
+
+const CondsIcon = <Conds/>;
+const StimIcon  = <StimTarget />;
+ 
 var PythonControlledCapability = require('../../../../../js/communication/geppettoJupyter/PythonControlledCapability');
 var PythonControlledTextField = PythonControlledCapability.createPythonControlledControl(TextField);
+var PythonControlledListComponent = PythonControlledCapability.createPythonControlledControl(ListComponent);
+var PythonMethodControlledSelectField = PythonControlledCapability.createPythonControlledControlWithPythonDataFetch(SelectField);
```

Comment:
@rodriguez-facundo Is this used anywhere?

Reply:
the first one is used to add multiple sections and the second one is used to select the stimSource that will be used from the list of stimSources previously created.

Reply:
You are right, my bad!